coalib/bearlib/aspects/meta.py

```diff
@@ -72,3 +73,23 @@ class Sub(subcls, aspectbase, metaclass=aspectclass):
 
     def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
+
+
+def issubaspect(aspect, subaspect):
+    """
+    This function checks whether or not ``subaspect`` is a subaspect of
+    ``aspect``.
+    """
+    if not isinstance(aspect, aspectclass):
+        raise TypeError(
+            "{} is not an aspectclass' instance".format(
+                type(aspect).__name__))
+    if not isinstance(subaspect, aspectclass):
+        raise TypeError(
+            "{} is not an aspectclass' instance".format(
+                type(subaspect).__name__))
+    if (aspect.__qualname__ in subaspect.__qualname__
+            and aspect.__qualname__ != subaspect.__qualname__):
```

Comment:
`subaspect.__qualname__.startswith(aspect.__qualname__)` would be a bit clearer.

BTW: You could also recursively check the `.parent` attribute. But that's just a matter of taste ;)